pokemongo_bot/__init__.py

```diff
@@ -322,7 +330,7 @@ def catch_pokemon(self, pokemon):
         return_value = worker.work()
 
         if return_value == PokemonCatchWorker.BAG_FULL:
-            worker = InitialTransferWorker(self)
+            worker = TransferWorker(self)
```

Comment:
The TransferWorker also handles parts of the transfer task that happens after post-capture